src/numberGenerator.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const MAX_NUMBER = 10;
```

Comment:
don't need to duplicated MAX_NUMBER, because you just import and export this variable